docs/testing.md

```diff
@@ -220,7 +220,7 @@ stdout. In the future there will be support for `lcov` output too.
 
 ```
 $ git clone git@github.com:denosaurs/deno_brotli.git && cd deno_brotli
-$ deno test --coverage --unstable
+$ deno test --coverage coverage_dir --unstable
```

Comment:
Definition in `flags.rs` says `--coverage` requires `=` to pass value

Reply:
Oops indeed that is out of date.